### PR TITLE
Development/major in details

### PR DESCRIPTION
**Pull request by apoalmadani**
mentees list of most recent majors shows on the participant details

### CODE REVIEW

File: app/views/participants/_details.html.erb [hunk 1]
```diff
@@ -20,6 +20,7 @@
 			<%= link_to "#{@participant.high_school.name} #{@participant.grad_year}", 
 					high_school_cohort_path(@participant.high_school, (@participant.grad_year || 0)), 
 					:class => 'redirect-link' unless @participant.high_school.nil? %>
+
```
Comment (mitchellharper12) [inline app/views/participants/_details.html.erb]: Could you remove this blank line and the extra whitespace on line 66, por favor? It's not a big issue, but lots of little places with extra whitespace can add up quickly.

File: app/views/participants/_details.html.erb [hunk 2]
```diff
@@ -52,10 +53,17 @@
 					<div class="small light">
 						<%= "#{enrollment.enrollment_status_name}, " unless enrollment.enrollment_status_name.blank? %>
 						<%= raw [enrollment.began_on.try(:to_s, :month_year), enrollment.ended_on.try(:to_s, :month_year)].compact.join('&ndash;') %>
+						<br/>
+						<% list_of_majors = enrollment.majors_list 
+						    majors = list_of_majors.split(", ")
+						    if majors.first.nil? || majors.first.downcase == "NOT APPLICABLE".downcase
+						    	list_of_majors = ""
+						    end %>
+						    <%=  list_of_majors %>
```
Comment (mitchellharper12) [inline app/views/participants/_details.html.erb]: Also this line could be moved over a few spaces to keep the indenting consistent.

**Comment by mitchellharper12** [comment]
@mattharris5 Does the logic for showing the current major look OK to you/is there any other feedback you could offer?

**Comment by mattharris5** [comment]
Anytime there's this much logic in the view, it's a warning that we aren't following MVC principles. This is logic that should live in the CollegeEnrollment model. If you take a look at CollegeEnrollment#majors_list, it already does some filtering there. How about just adding to that to take out your "NOT APPLICABLE" majors? Then you can just display `<%= enrollment.majors_list %>` and have a nice comma-separated list in the view. What do you think, @apoalmadani?

**Pull request closed without merging**